uqcsbot/scripts/advent.py

```diff
@@ -0,0 +1,61 @@
+from uqcsbot import bot, Command
+from requests.exceptions import RequestException
+from typing import Dict, List
+import requests
+
+LEADERBOARD_URL = 'https://adventofcode.com/2018/leaderboard/private/view/'
+LEADERBOARD_CODE = '246889'
+# Session ID goes here
+SESSION_ID = '' 
```

Comment:
This can be stored in an environment variable, e.g.:
os.environ['AOC_SESSION_ID']
I can modify the supervisord config to store a session ID, though I expect they will expire at some point (though maybe not until AoC finishes)